### PR TITLE
Adding 'manifest XML' here to clarify for React

### PR DESCRIPTION
**Pull request by mscharlock**
The React tutorial also creates a manifest.json file in a public folder which seemed like it should be the manifest file to me as a first go through of this. However, turns out they want you to select my-office-addin-manifest.xml, which I totally didn't even see because of the name. So, I tried to clarify here by stating explicitly that it's an XML file. I don't think this problem happens in the others, though I haven't done Vue or Angular.

### CODE REVIEW

**Comment by o365devx** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/mscharlock/office-js-docs-pr/blob/Michelle-ExcelAddins/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :warning:Warning | | [Details](#user-content-docs%2Ftesting%2Fcreate-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)

<a name="docs%2Ftesting%2Fcreate-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md"></a>
### [docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/mscharlock/office-js-docs-pr/blob/Michelle-ExcelAddins/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/../../../outlook/add-ins/sideload-outlook-add-ins-for-testing).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808071848446116-296%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=TFqLjOYJhA5poh%2B3NXxGBhHjWhJHLVXOG5cLTC9Ve2w%3D&st=2018-08-07T18%3A45%3A09Z&se=2018-09-07T18%3A50%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :x: Validation status: errors


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-)

<a name=""></a>
### []()
 - **[Error]** Cannot sync git repo to specified state. Please make sure the pull request is mergeable. Details: Git merge results in conflict, repository remote url: https://github.com/OfficeDev/office-js-docs-pr, working directory: W:\sndr-s, target branch: master, target branch commit: 7f28ee8811f19bb7ae7b1020badbffe132c192dc, source branch: 81e71bd04e234de2261872d2533de966/Michelle-ExcelAddins, source branch commit: .

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808072102355067-296%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=y6IU9Pk1kEizOP1aB5XbiDmjLOHHhVZbxf7ZnuXqHJY%3D&st=2018-08-07T20%3A57%3A46Z&se=2018-09-07T21%3A02%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/mscharlock/office-js-docs-pr/blob/Michelle-ExcelAddins/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :warning:Warning | | [Details](#user-content-docs%2Ftesting%2Fcreate-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)

<a name="docs%2Ftesting%2Fcreate-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md"></a>
### [docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/mscharlock/office-js-docs-pr/blob/Michelle-ExcelAddins/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md)
 - **Line 18**: **[Warning]** Invalid file link:(~/../../../outlook/add-ins/sideload-outlook-add-ins-for-testing.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808072103357106-296%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=znEbWssEZlrR4G4is5ZNnoyqq2LLhpB%2B00V1mk79tO4%3D&st=2018-08-07T20%3A59%3A48Z&se=2018-09-07T21%3A04%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md [hunk 1]
```diff
@@ -15,7 +15,8 @@ This article applies only to testing a Word, Excel, or PowerPoint add-ins on Win
 
 - [Sideload Office Add-ins in Office Online for testing](sideload-office-add-ins-for-testing.md)
 - [Sideload Office Add-ins on iPad and Mac for testing](sideload-an-office-add-in-on-ipad-and-mac.md)
-- [Sideload Outlook Add-ins for testing](https://docs.microsoft.com/outlook/add-ins/sideload-outlook-add-ins-for-testing)
+- [Sideload Outlook Add-ins for testing](../../../../outlook/add-ins/sideload-outlook-add-ins-for-testing.md)
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md]: this undoes my warning fix...

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md](https://github.com/mscharlock/office-js-docs-pr/blob/Michelle-ExcelAddins/docs/testing/create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C8%5C7%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201808072211368922-296%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=XKgD0PujIMQ2IYtKR01uHRnpn4foJXqnxzmxw6HiW24%3D&st=2018-08-07T22%3A07%3A47Z&se=2018-09-07T22%3A12%3A47Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**